Prioritize/DotNet/Single Project/azure-pipelines_windows.yml

```diff
@@ -34,6 +34,7 @@ steps:
       set WS_RESOLVEALLDEPENDENCIES=false
       set WS_NUGET_RESOLVEDEPENDENCIES=true
       set WS_NUGET_RESTOREDEPENDENCIES=true
+      set WS_FILESYSTEMSCAN=false
       set WS_EXCLUDES=**\build\** **\test**
```

Comment:
Missing a slash after `**\test`
BTW, I tend to specify includes/excludes params with forward slashes even when running on Windows, not only because Windows can already interpret it, but mostly because *JVM* needs to parse it, and backslashes could lead to unexpected results, depending on the shell agent/interpreter.